src/App.tsx

```diff
@@ -8,55 +8,148 @@ import { PostsList } from './components/PostsList';
 import { PostDetails } from './components/PostDetails';
 import { UserSelector } from './components/UserSelector';
 import { Loader } from './components/Loader';
+import { getUsers } from './api/users';
+import { User } from './types/User';
+import { getPosts } from './api/posts';
+import { Post } from './types/Post';
+import { getComments } from './api/commets';
+import { Comment } from './types/Comment';
 
 export const App: React.FC = () => {
+  const [userSelect, setUserSelect] = useState<User[]>([]);
+  const [selectedUser, setSelectedUser] = useState<User | null>(null);
+
+  const [isPostError, setIsPostError] = useState<boolean>(false);
+  const [isPostsLoading, setIsPostsLoading] = useState<boolean>(false);
+  const [userPosts, setUserPosts] = useState<Post[]>([]);
+
+  const [postIdComments, setPostIdComments] = useState<number | null>(null);
+  const [isCommentsLoading, setIsCommentsLoading] = useState<boolean>(false);
+  const [postComments, setPostComments] = useState<Comment[]>([]);
+  const [isCommentError, setIsCommentError] = useState<boolean>(false);
+
+  const [isNewComment, setIsNewComment] = useState<boolean>(false);
+
+  const fetchUsers = useCallback(async () => {
+    try {
+      const usersFromServer = await getUsers();
+
+      setUserSelect(usersFromServer);
+    } catch (error) {
+      setIsPostError(true);
+    }
+  }, [userSelect]);
+
+  const fetchPosts = useCallback(async (id: number) => {
+    try {
+      const postsFromServer = await getPosts(id);
+
+      setUserPosts(postsFromServer);
+    } catch (error) {
+      setIsPostError(true);
+    }
+  }, [userPosts]);
+
+  const fetchComments = useCallback(async (id: number) => {
+    try {
+      const commentsFromServer = await getComments(id);
+
+      setPostComments(commentsFromServer);
+    } catch (error) {
+      setIsCommentError(true);
+    }
+  }, [postComments]);
+
+  useEffect(() => {
+    fetchUsers();
+  }, []);
```

Comment:
Include any reactive values that are used inside useEffect into its deps array.
https://stackoverflow.com/questions/58866796/understanding-the-react-hooks-exhaustive-deps-lint-rule

Reply:
I cant understand what I can add? I did track a traffic flow and through all app only 1 users fetch happen, not any more, what I need to resolve by adding reactive value to useEffect dependencies?